--- a/tickettemplate/htdocs/tt_newticket.js	2016-02-17 05:10:05.000000000 +0100
+++ b/tickettemplate/htdocs/tt_newticket.js	2016-02-17 08:07:15.000000000 +0100
@@ -43,6 +43,12 @@
               elem.selected = true;
             }
           }
+        } else if (targetElem.type == 'hidden') {
+          // for SubcomponentsPlugin
+          var comp_parts = ticketType[field].split('/');
+          for (var i = 1; i <= comp_parts.length; i++) {
+              $('#component-selector1-' + i).val(comp_parts[i - 1]).change();
+          }
         } else {
           targetElem.value = ticketType[field];
         }
